full manual workflow test is flaky, at least on macos

```
Traceback (most recent call last):
  File "/Users/runner/work/ert/ert/tests/unit_tests/gui/test_full_manual_update_workflow.py", line 43, in handle_dialog
    assert isinstance(messagebox, QMessageBox)
AssertionError: assert False
 +  where False = isinstance(<ert.gui.ertwidgets.statusdialog.StatusDialog object at 0x1494b1090>, QMessageBox)
```
seen here: https://github.com/equinor/ert/actions/runs/6413437015/job/17412409049?pr=6241

--- a/tests/unit_tests/gui/test_full_manual_update_workflow.py
+++ b/tests/unit_tests/gui/test_full_manual_update_workflow.py
@@ -39,6 +39,9 @@ def test_that_the_manual_analysis_tool_works(
 
         # Click on "Run" and click ok on the message box
         def handle_dialog():
+            qtbot.waitUntil(
+                lambda: isinstance(QApplication.activeWindow(), QMessageBox)
+            )
             messagebox = QApplication.activeWindow()
             assert isinstance(messagebox, QMessageBox)
             ok_button = messagebox.button(QMessageBox.Ok)

